services/auth/src/gcp.js

```diff
@@ -0,0 +1,97 @@
+const builder = require('./api');
+const {google} = require('googleapis');
+
+builder.declare({
+  method: 'get',
+  route: '/gcp/credentials/:projectId/:serviceAccount',
```

Comment:
In a sense, `projectId` will always be the same since the `serviceAccount` that auth runs as is only in a single account. We should probably either remove this from the route or allow passing in a set of different application credentials in config, one per project. Does that make sense?

Reply:
Yes, done.